packages/ckeditor5-ckfinder/docs/features/ckfinder.md

```diff
@@ -14,6 +14,10 @@ This feature allows you to insert images as well as links to files into the rich
 	Check out the comprehensive {@link features/image-upload Image upload overview} to learn about other ways to upload images into CKEditor 5.
 </info-box>
 
+<info-box>
```

Comment:
Since there are already 2 other info boxes in this section, maybe it'd be better to just move this information to a normal sentence in the first paragraph?

Reply:
Please do.